src/Engine/Manager/SoundManager.hpp

```diff
@@ -101,10 +99,16 @@ class SoundManager {
         /// Remove all killed components.
         ENGINE_API void ClearKilledComponents();
 
-        /// Add work to streaming thread.
+        /// Load audio from file.
+        /**
+         * @param dataHandle %DataHandle to load.
```

Comment:
We may want to remove percentage character to allow linking to the type in the documentation.